jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletTester.java

```diff
@@ -39,6 +40,51 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * <p>ServletTester should not be used.</p>
+ * <p>ServletTester is a just a wrapper around a {@link ServletContextHandler},
+ * with a {@link LocalConnector} to accept HTTP/1.1 requests, so there is no
+ * value that this class adds to already existing classes.</p>
+ * <p>Replace its usages with:</p>
+ * <pre>
+ * Server server = new Server();
+ * LocalConnector connector = new LocalConnector(server);
+ * server.addConnector(connector);
+ * ServletContextHandler context = new ServletContextHandler(server, "/contextPath");
+ * // Configure the context here.
+ * server.start();
+ * </pre>
+ * <p>You can configure the context by adding Servlets and Filters, attributes,
+ * etc. even after it has been started.
+ * Use {@link HttpTester} and {@link LocalConnector} to make HTTP/1.1 requests,
+ * in this way:</p>
+ * <pre>
+ * // Generate the request.
```

Comment:
Perhaps show two examples.  One using LocalConnector directly with strings and then show HttpTester usage.

Reply:
actually never mind...